nmpolicy/internal/resolver/resolver_test.go

```diff
@@ -370,13 +372,16 @@ base-iface-routes:
         table-id: 254
 `,
 		}
-		runTest(t, testToRun)
+		runTest(t, &testToRun)
 	})
 }
 
 func testFilterInvalidTypeOnPath(t *testing.T) {
 	t.Run("Filter invalid type on path", func(t *testing.T) {
 		testToRun := test{
+			captureExpressions: `
+invalid-path-type: "01234567"
```

Comment:
Can you please use the real capture expression `invalid-path-type: "interfaces.ipv4.address==10.244.0.1"` instead of `"01234567"`?
For me it was hard to follow and see it the `^` place makes sense when using `"01234567"`.

Reply:
Problem with that is that make more difficult to put correct pos at AST.

Reply:
I understand, but I believe it worth the effort since it will make the test much more readable.